src/components/BookForm.jsx

```diff
@@ -80,11 +72,12 @@ function BookForm({ url }) {
       </div>
       <div className="row">
         <div className="col-12">
-          <h2 className="text-secondary my-3 text-uppercase">Add New Book</h2>
-          <form>
+          <hr className="py-2" />
+          <h2 className="text-secondary my-3 text-uppercase fs-4 fw-bolder">Add New Book</h2>
+          <form className="col-form-label-lg">
             <label className="px-3" htmlFor="title">
               <input
-                className="px-2 py-1 rounded border border-secondary"
+                className="px-2 py-1 rounded border border-secondary input-title"
                 type="text"
```

Comment:
- [ ] According to the project requirement you should use the **EXACTLY** the same as the one provided on Zeplin. See the design below and adjut the spaing and also, the same color.
### Your design
![his](https://github.com/PabloBona/bookstore-react-app/assets/81727988/d5b4072b-9e23-4b60-af8d-09ad49b5c3fd)



### Original design
![original](https://github.com/PabloBona/bookstore-react-app/assets/81727988/3d62a524-726f-4fa4-b0fd-a3c2d2f99b42)

Reply:
Can you give me the link where it says that the design has to be exactly the same? that's not what my requirements say...
/
![Screenshot_20230721-140908](https://github.com/PabloBona/bookstore-react-app/assets/110569486/e3480ccf-fe39-49da-ad68-7dd47483cd36)
@roshan-bajgain

Reply:
Hello @PabloBona, It says the `same styling` to the zipline design as it helps you when you work for your client to exactly do the same design as they want. Also, It helps when you go through the job search process. So please try to follow the exact design.

Reply:
I think that there is a distinction between the phrases 'exactly the same' and 'should match' in the mentioned requirements. I would like to receive guidance on identifying specifically where the word 'exactly' is mentioned in the requirements. Your assistance in this matter would be greatly appreciated.
@roshan-bajgain

Reply:
please adjust the spacing and also make sure that color matches the design then after that you can ask for another review.

Reply:
@roshan-bajgain
The power went out all day, just now I was able to make the corrections